apps/src/JsInterpreterLogger.js

```diff
@@ -0,0 +1,59 @@
+/** @file Observes a JSInterpreter and logs to the designated browser console. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
```

Comment:
Why do we have such a different set of jshint rules for this file? Can they come with a short comment explaining why they're different?